protos/sources/vega/vega.proto

```diff
@@ -1697,7 +1697,7 @@ message ReferralProgram {
   // second element will give Tier 2, and so on. Determines the level of
   // benefit a party can expect based on performance criteria.
   repeated BenefitTier benefit_tiers = 3;
-  // Timestamp as Unix time in seconds after which when the current epoch ends, the
+  // Timestamp as Unix time in seconds, after which when the current epoch ends, the
```

Comment:
same question about seconds/nanoseconds

Reply:
As above, the timestamp may be provided as seconds from core, the timestamp returned from GQL API will have nanosecond precision, even though it's all 0's